### DISCUSSION TITLE
"install-path" in the wsl --import command. What is it?

### DISCUSSION THREAD

**Discussion by o-alquimista** [dorssel/usbipd-win#628] [Q&A] [answered]
From https://github.com/dorssel/usbipd-win/wiki/WSL-support#building-your-own-usbip-enabled-wsl-2-kernel:
> Import new distribution with current distribution as base.
> `wsl --import wsl2-usbip <install-path> <temporary-path>\wsl2-usbip.tar`

What do I put on `<install-path>`? I don't know any installation path, I was never asked for one. There's no description of this in `usbipd-win` or in the Microsoft documentation for the `wsl --import` command.

**Comment by dorssel** [comment, marked as answer]
You may choose it yourself. It is where the distro will store its data.

**Reply by o-alquimista** [reply]
I just want the default. What do you use? Is it where the `ext4.vhdx` file is?

e.g. `C:\Users\myuser\AppData\Local\Packages\CanonicalGroupLimited.Ubuntu_79rhkp1fndgsc\LocalState\ext4.vhdx`

**Reply by o-alquimista** [reply]
I think I understand it now. We are creating a "throw-away" distribution just to build the kernel. I can just store it in my home folder then.

**Reply by dorssel** [reply]
Yes, exactly.

**Reply by biggestsonicfan** [reply]
I'm fairly new to WSL2, but more than a novice user of both Windows and Linux and found the instructions fairly confusing at this part. Very glad to see this was clarified in discussions but perhaps a minor note or clarification in the wiki would be beneficial. 😃
